neurogym/config/components/trigger.py

```diff
@@ -0,0 +1,28 @@
+from typing import Literal
+
+from pydantic import model_validator
+
+from neurogym.config.base import ConfigBase
+
+
+class TriggerConfig(ConfigBase):
+    """Generic configuration for components that run by 'trials' or 'steps'.
+
+    Attributes:
+        trigger: Either 'trials' or 'steps'.
+        value: Positive integer corresponding to the trigger.
+    """
+
+    trigger: Literal["trials", "steps"]
+    value: int
```

Comment:
Would `interval` instead of `value` be more descriptive here? As in 'an event occurs every 10 steps' => the 'interval between successive events is 10 steps'.